app/views/marketing/marketing.html.erb

```diff
@@ -8,7 +8,7 @@
 		<p class="hero__copy">
 			HuBoard is built from the ground up using the GitHub public API. HuBoard issues are GitHub issues, you will never have to deal with synchronization problems. Keep issues where they belong, in the repository with your code!
 		</p>
-		<a href="/login" class="btn hero__btn">Sign up for a 14 day free trial <i class="icon icon--lt"></i></a>
+		<a href="/login/public" class="btn hero__btn">Try HuBoard for free! <i class="icon icon--lt"></i></a>
```

Comment:
Even if we end up keeping the header change to `/login/public`, it seems wise to me that "Try Us" type links would still go through `/login` to give us a chance to clarify what the following GitHub auth screen means.